src/Generator/ValueGenerator.php

```diff
@@ -143,7 +149,7 @@ public function deleteConstant($constant)
     /**
      * Return constant list
      *
-     * @return ArrayObject
+     * @return SplArrayObject|StdlibArrayObject
```

Comment:
should this return a mixed type, or should it always return a SPL ArrayObject by converting StdLib ones ?